pwndbg/dbg/__init__.py

```diff
@@ -765,6 +765,78 @@ def keys(self) -> List[str]:
         # if there is a better debugger-specific way to do this.
         return [field.name for field in self.fields()]
 
+    def enum_member(self, field_name: str) -> int | None:
+        """
+        Retrieve the integer value of an enum member.
+
+        It returns:
+        - integer value, when found field
+        - returns None, If the field does not exist
+        """
+        if self.code != TypeCode.ENUM:
+            raise ValueError("only enum supported")
+
+        return next((f.enumval for f in self.fields() if f.name == field_name), None)
+
+    def _offsetof(
+        self, field_name: str, *, base_offset_bits: int = 0, nested_cyclic_types: List[Type] = None
+    ) -> int | None:
+        NESTED_TYPES = (TypeCode.STRUCT, TypeCode.UNION)
+        struct_type = self
+        if nested_cyclic_types is None:
+            nested_cyclic_types = []
+
+        if struct_type.code == TypeCode.TYPEDEF:
+            struct_type = struct_type.strip_typedefs()
+
+        if struct_type.code == TypeCode.POINTER:
+            struct_type = struct_type.target().strip_typedefs()
+
+        if struct_type.code not in NESTED_TYPES:
```

Comment:
```suggestion
        elif struct_type.code not in NESTED_TYPES:
```

Reply:
<img width="441" alt="image" src="https://github.com/user-attachments/assets/9ce66fe8-3322-4ab8-b8f0-7357cb025b23" />
we are changing same variable, and it must be just `if` not `elif`